lib/nerves_wpa_supplicant/messages.ex

```diff
@@ -150,8 +150,14 @@ defmodule Nerves.WpaSupplicant.Messages do
     resp
     |> String.split("\n", trim: true)
     |> List.foldl(%{}, fn pair, acc ->
-      [key, value] = String.split(pair, "=")
-      Map.put(acc, String.to_atom(key), kv_value(String.trim_trailing(value)))
+      case String.split(pair, "=") do
+        [key, value] ->
+          acc
+          |> Map.put(String.to_atom(key), kv_value(String.trim_trailing(value)))
+
+        _ ->
```

Comment:
Would it be useful to log the offending string?

Reply:
I think it would be. I'll add this

Reply:
I'm not sure which network it's balking at. Here are a couple:

```
DIRECT-0F-HP OfficeJetPro 8710
better-than-going-to-the-library
```

I can test it out tonight. Is there a way to tell from the log message which one is crashing it?

Reply:
@aaronrenner I added a log message. I'd appreciate if you can try this out. 
It's likely some P2P device, such as a Roku, printer, TV etc

Reply:
Cool, thanks! I'll try it out this evening or first thing tomorrow morning.